todo_app/view_model.py

```diff
@@ -0,0 +1,23 @@
+class ViewModel:
+  def __init__(self, lists):
+    self._lists = lists
+
+  @property
+  def lists(self):
+    return self._lists
+  
+  @property
+  def items(self):
+    return self._lists

```

Comment:
Not sure if we're mismatching language or this is a typo, but we were assuming `items` meant individual to-dos, rather than lists of those. So this might be confusing!